src/ViewInterface.php

```diff
@@ -100,6 +100,13 @@ public function withLocale(string $locale): static;
      */
     public function getBasePath(): string;
 
+    /**
+     * Gets the context instance, or `null` if no context has been set.
+     *
+     * @return ViewContextInterface|null The context instance, or `null` if no context has been set.
+     */
+    public function getContext(): ?ViewContextInterface;
```

Comment:
That is BC breaking and requires major release.

Is there no other way to get the data in https://github.com/yiisoft/view-twig/pull/61/files#diff-6c70c51c86f360f182ffdb6d3edd3b48ba745382a95189bd4cb933d34805f128R33 ?

Reply:
https://github.com/yiisoft/view/pull/226/commits/e742f6722558fbaa610caec6a2f68c4ac2461602